api/src/main/java/org/openmrs/module/locationbasedaccess/common/Constants.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.locationbasedaccess.common;
+
+public class Constants {
+
+    public static final String ACCESS_LOCATION_PERSON_ATTRIBUTE_TYPE_NAME = "accessControlLocation";
```

Comment:
What are you using this for?

Reply:
To create the person attribute for the first time (it will keep the person attribute name as a constant) .

Reply:
Is that required as part of our first use case?

Reply:
yes, Otherwise I can't create a new person attribute, and Can't track that personAttribute in the methods.

Reply:
@dkayiwa  As we are dealing with very very first use case, I'm removing this constants as well :+1: